webrtc.html

```diff
@@ -2389,9 +2389,9 @@
           takes a dictionary argument, <var>candidateInitDict</var>, whose content is used
           to initialize the new
           <a class="idlType" href="#idl-def-RTCIceCandidate"><code>RTCIceCandidate</code></a>
-          object. If either <var>sdpMid</var> or <var>sdpMLineIndex</var> is not present in
+          object. If the <var>sdpMLineIndex</var> is not present in
           <var>candidateInitDict</var>, the corresponding attribute will be initialized to
-          <code>null</code>. If neither is present, a <code>TypeError</code> exception
+          <code>null</code>. If <var>sdpMid</var> is not present, a <code>TypeError</code> exception
           will be thrown.</dd>
```

Comment:
> On Mar 24, 2016, at 3:27 AM, Adam Bergkvist <notifications@github.com> wrote:
> 
> In webrtc.html:
> 
> >            <var>candidateInitDict</var>, the corresponding attribute will be initialized to
> > -          <code>null</code>. If neither is present, a <code>TypeError</code> exception
> > +          <code>null</code>. If <var>sdpMid</var> is not present, a <code>TypeError</code> exception
> >            will be thrown.</dd>
> 
> It it safe to require sdpMid here? We accept remote descriptions without mid and generate one in that case.
> 
> This behaviour (throw if not present) can also be expressed with the "required" WebIDL keyword on the member. And should we keep sdpMLineIndex if we require sdpMid?
> 
> —


I don't think we can require it here because it should not be retuned if it was not in remote description as that would cause something to fetched it to think the remote side supported mid when it does not. 

We have to keep the mLineIndiex